### PR TITLE
add repository structure doc and update root readme

### PR DESCRIPTION
**Pull request by kenakamu**
I added detail repository structure information in docs and add link to root Readme.

### CODE REVIEW

File: docs/process/RepositoryStructure.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Repository Structure
+
+This repo contains sample code and definition of Azure DevOps pipelines for CI/CD. These pipelines run in this [Azure DevOps project](https://dev.azure.com/cse-manufacturing/MLOpsManufacturing/_build?view=folders). Azure ML pipelines run in the Azure environment deployed using the Azure DevOps pipelines.
+
+# Folder structure
+
+```bash
+├─ common # contains code common across samples
+│    ├─ infrastructure # code to provision Azure resources to run the samples
+│    └─ pipeline-trigger
```
Comment (liupeirong) [inline docs/process/RepositoryStructure.md]: Add an inline comment here?

File: README.md [hunk 2]
```diff
@@ -11,22 +11,7 @@ This repo contains samples of machine learning (ML) projects we have often seen
...
-        - .devcontainer # VSCode dev container you can optionally use for development
-        - devops_pipelines # Azure DevOps CI/CD pipeline definition
-        - local_development # scripts for creating a local dev environment without having to have a VSCode dev container
-        - ml_model # code for building the ML model
-        - ml_service # code for building ML pipelines
-        - README.md # explains what the sample is demonstrating and how to run it
-    - <sample 2>
-    - ...
-```
+See the detail inforamtion at [Repository Structure](./docs/process/RepositoryStructure.md)
```
Comment (liupeirong) [inline README.md]: just a bit different wording: See details on [Repository Structure](...)

File: docs/process/RepositoryStructure.md (hunk 1 above)
Comment (kenakamu) [reply docs/process/RepositoryStructure.md]: i deleted this as i didn't commit the function app yet

File: README.md (hunk 2 above)
Comment (kenakamu) [reply README.md]: done :)

**Comment by aykhara** [comment]
Thank you @kenakamu, it looks good to me.

**Review by liupeirong — APPROVED** [review]

**Pull request merged**